### PR TITLE
Add command to control braking on individual axis

### PR DESCRIPTION
**Pull request by Jeff-Ciesielski**
Closes #65

### CODE REVIEW

File: stepper.c [hunk 1]
```diff
@@ -298,6 +301,16 @@ void st_wake_up()
   }
 }
 
+void st_stop_shutdown_timer(void)
+{
+  st_shutdown_start = 0;
+}
```
Comment (StephanCoetzee) [inline stepper.c:312]: Will it not be more efficient if we make this into a macro? That way it doesn't get pushed onto and popped off the call stack when it's called:
`#define st_stop_shutdown_timer() (st_shutdown_start = 0)`

File: main.c [hunk 2]
```diff
@@ -41,7 +41,10 @@
 #include "signals.h"
 
 // Declare system global variable structure
-system_t sys;
+system_t sys = {
+  .lock_mask = STEPPERS_LONG_LOCK_MASK,
+};
+
```
Comment (StephanCoetzee) [inline main.c:47]: Why is this not in `sys_init()`?

File: system.c [hunk 3]
```diff
@@ -295,6 +295,49 @@ uint8_t system_execute_line(char *line)
           if (line[++char_counter] != 0)
             return(STATUS_INVALID_STATEMENT);
           break;
+        case 'B': // Turn braking on or off
+	{
```
Comment (StephanCoetzee) [inline system.c]: This brace should be on the previous line according to the kernel style guide.

Comment (Jeff-Ciesielski) [reply system.c]: Not in a case statement.

File: stepper.c (hunk 1 above)
Comment (Jeff-Ciesielski) [reply stepper.c:312]: With -Os, this will likely be inlined anyway.

File: system.c (hunk 3 above)
Comment (Jeff-Ciesielski) [reply system.c]: I will fix the indentation though :). My editor decided a tab was appropriate here...

File: main.c (hunk 2 above)
Comment (Jeff-Ciesielski) [reply main.c:47]: Because it needs to happen before the steppers are initialized.

**Comment by mdicicco** [comment]
you've got a conflict now

**Comment by Jeff-Ciesielski** [comment]
Fixed

**Comment by StephanCoetzee** [comment]
Conflict again.

**Comment by Jeff-Ciesielski** [comment]
Rebased

**Comment by mdicicco** [comment]
:+1:

**Pull request merged**